src/App/Link.cpp

```diff
@@ -51,6 +54,80 @@ using namespace App;
 using namespace Base;
 namespace bp = boost::placeholders;
 
+typedef boost::iterator_range<const char*> CharRange;
```

Comment:
Is this just a `std::span`?

Reply:
More like string_view I guess. It's mainly used with boost string_algo.